### ISSUE TITLE
'pod trunk push' should check email before building podspec

### ISSUE THREAD

**Issue by wjmelements** [CocoaPods/CocoaPods#7923]
It's a shame to complete the long validation process only to get that you have the wrong email registered. I believe that the email check should be done sooner.

**Comment by orta** [comment]
Yeah, sounds like a good idea, you'll need to add a route to trunk for verifying and update the cocoapods-trunk gem 👍
